src/engine/client/input.cpp

```diff
@@ -61,6 +63,33 @@ void CInput::Init()
 	// FIXME: unicode handling: use SDL_StartTextInput/SDL_StopTextInput on inputs
 
 	MouseModeRelative();
+
+	if (!SDL_WasInit(SDL_INIT_JOYSTICK))
+	{
+		if (SDL_InitSubSystem(SDL_INIT_JOYSTICK) < 0) {
+			dbg_msg("joystick", "unable to init SDL joystick: %s", SDL_GetError());
+			return;
+		}
+	}
+
+	m_Joystick = NULL;
```

Comment:
Teeworlds doesn't like `NULL`, I think you're supposed to use 0 (or 0x0). There are two other instances of that.